src/api/controller/MyController.hpp

```diff
@@ -78,31 +65,31 @@ class MyController : public oatpp::web::server::api::ApiController {
 
     auto responseDto = bakcyl::api::LoginResponseDto::createShared();
 
-    if (json && json->username && json->password) {
-      std::string token = auth.authenticateUser(json->username, json->password);
-
-      bool loginSuccess = false;
-      if (token != "") loginSuccess = true;
-      responseDto->success = loginSuccess;
-
-      if (loginSuccess) {
-        auto response = createDtoResponse(Status::CODE_200, responseDto);
-        response->putHeader("Authorization", "Bearer " + token);
-        return response;
-      } else {
-        return createDtoResponse(Status::CODE_403, responseDto);
-      }
-    } else {
+    if (!json && !json->username && !json->password) {
       responseDto->success = false;
       return createDtoResponse(Status::CODE_400, responseDto);
     }
+
+    std::string token = auth.authenticateUser(json->username, json->password);
+
+    bool loginSuccess = false;
+    if (token != "") loginSuccess = true;
```

Comment:
use { }

Reply:
the comment applies to all changes, I don't want to add this comment in every place

Reply:
the commits [2897504](https://github.com/BakcylProgramowania/paragon_back/pull/90/commits/289750421b26bf40edfa9ae2b5b1172938460bf6) and [1f8ab40](https://github.com/BakcylProgramowania/paragon_back/pull/90/commits/1f8ab40be0974e8ba2c381e4c3aa4a1524529523) solved the issue